cocotb/share/makefiles/simulators/Makefile.xcelium

```diff
@@ -79,8 +79,8 @@ endif
 # which does not play nice with our multi language, so we supply GPI_EXTRA=cocotbvhpi if needed.
 
 #GPI_EXTRA:=
+GPI_ARGS = -loadvpi $(LIB_DIR)/libcocotbvpi:vlog_startup_routines_bootstrap
 ifeq ($(TOPLEVEL_LANG),verilog)
-    GPI_ARGS = -loadvpi $(LIB_DIR)/libvpi
```

Comment:
This looks unrelated?

Reply:
Actually this is very interesting. This line was doing nothing before. For some reason, `libvpi` would be loaded in any case.  Now since we do not have  `libvpi`  have to use this properly.

Reply:
This type of "very interesting" is what would really be great in the commit message